lib/g_searcher/search_result_urls.ex

```diff
@@ -0,0 +1,16 @@
+defmodule GSearcher.SearchResultURLs do
+  import Ecto.Query, warn: false
+
+  alias GSearcher.Repo
+  alias GSearcher.SearchResults.SearchResultURL
+
+  def get_by_search_result_id(search_result_id) do
+    SearchResultURL
+    |> where([sru], sru.search_result_id == ^search_result_id)
+    |> Repo.all()
+    |> case do
+      [] -> {:ok, []}
+      search_result_urls -> {:ok, search_result_urls}
```

Comment:
We can also do it like this
```suggestion
    search_result_urls = SearchResultURL
    |> where([sru], sru.search_result_id == ^search_result_id)
    |> Repo.all()
    
    {:ok, search_result_urls}
```